basic/src/test/java/org/jboss/hal/testsuite/test/runtime/server/ForceShutdownOperationTestCase.java

```diff
@@ -0,0 +1,195 @@
+        try {
+            new FinderNavigation(browser, DomainRuntimeEntryPoint.class)
+                    .step(FinderNames.BROWSE_DOMAIN_BY, FinderNames.SERVER_GROUPS)
+                    .step(FinderNames.SERVER_GROUP, testServerGroup)
+                    .step(FinderNames.SERVER, testServerName)
+                    .selectRow().invoke(FORCE_SHUTDOWN);
+            Console.withBrowser(browser).openedWindow(ConfirmationWindow.class).confirm();
+            Console.withBrowser(browser).waitUntilLoaded();
+            new ResourceVerifier(serverAddress, client).verifyAttribute(SERVER_STATE, "STOPPED");
+        } finally {
+            domainAdministration.reloadIfRequired();
+        }
+    }
+
+    @Test
+    @RunAsClient
+    public void testForceShutdownByHost() throws Exception {
+        try {
+            new FinderNavigation(browser, DomainRuntimeEntryPoint.class)
+                    .step(FinderNames.BROWSE_DOMAIN_BY, FinderNames.HOSTS)
+                    .step(FinderNames.HOST, defaultHost)
+                    .step(FinderNames.SERVER, testServerName)
+                    .selectRow().invoke(FORCE_SHUTDOWN);
+            Console.withBrowser(browser).openedWindow(ConfirmationWindow.class).confirm();
+            Console.withBrowser(browser).waitUntilLoaded();
+            new ResourceVerifier(serverAddress, client).verifyAttribute(SERVER_STATE, "STOPPED");
+        } finally {
+            domainAdministration.reloadIfRequired();
+        }
+    }
+
+    @Test
+    public void testForceShutdownInLoad() throws Exception {
+        Runnable loadServerTask = this::loadServer;
+        Thread task = new Thread(loadServerTask);
+        try {
+            task.start();
+            new FinderNavigation(browser, DomainRuntimeEntryPoint.class)
+                    .step(FinderNames.BROWSE_DOMAIN_BY, FinderNames.HOSTS)
+                    .step(FinderNames.HOST, defaultHost)
+                    .step(FinderNames.SERVER, testServerName)
+                    .selectRow().invoke(FORCE_SHUTDOWN);
+            Console.withBrowser(browser).openedWindow(ConfirmationWindow.class).confirm();
+            Console.withBrowser(browser).waitUntilLoaded();
+            new ResourceVerifier(serverAddress, client).verifyAttribute(SERVER_STATE, "STOPPED");
+        } finally {
+            task.interrupt();
+        }
+    }
+
+
+    private void loadServer() {
+        final String serverUrl = String.format("http://127.0.0.1:%d/%s%s", SERVER_PORT,
+                DEPLOYMENT_NAME, InfiniteServlet.URL_PATTERN);
+        try (CloseableHttpClient client = HttpClientBuilder.create().build()) {
+            while (!Thread.currentThread().isInterrupted()) {
+                Thread.sleep(100);
+                HttpGet httpGet = new HttpGet(serverUrl);
+                CloseableHttpResponse response = client.execute(httpGet);
+                HttpEntity entity = response.getEntity();
+                LOGGER.debug(EntityUtils.toString(entity));
```

Comment:
Wouldn't it be more beneficial to just check/log response code? Instead of whole response? Also if you are reading it the entity, you don't need to consumeQuietly the entity again as you've already read it.